app.test.js

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, it } from 'vitest';
+import app from './app';
+
+describe('Get health route', () => {
+  it('checks that status is 200', async () => {
+    const res = await app.inject({
+      method: 'GET',
+      url: '/health',
+    });
+
+    expect(res.statusCode).toBe(200);
+    expect(res.body).toEqual('ok');
+  });
+
+  it('checks fot 404 status when path is wrong', async () => {
```

Comment:
*`for`

Reply:
`path` is not the right term... maybe `route`?

Reply:
done